Makefile

```diff
@@ -2,6 +2,11 @@ export GOROOT=$(shell go env GOROOT)
 export GOFLAGS=-mod=vendor
 export GO111MODULE=on
 
+ES_PROXY_MAJOR="1"
```

Comment:
Is there any reason not to add these directly to the Dockerfile.in?  There is nothing about the changes where we are calculating version

Reply:
updated